### PR TITLE
TASK: Update for Neos 7.0 compatibility

### PR DESCRIPTION
**Pull request by dfeyer**

### CODE REVIEW

**Comment by dfeyer** [comment]
@lorenzulrich If you have time to test this one, early WIP

**Comment by lorenzulrich** [comment]
@dfeyer Sorry for the late reply, only seen this now. I will test this in Q1 2022 during an upgrade.

File: Classes/Aspect/EventLogAspect.php [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <?php
+declare(strict_types=1);
+
 namespace Ttree\ContentRepositoryImporter\Aspect;
 
+use Psr\Log\LoggerInterface;
```
Comment (markusguenther) [inline Classes/Aspect/EventLogAspect.php]: You could use something like this:

```use Psr\Log\LoggerInterface as SystemLoggerInterface;```

File: Classes/Aspect/EventLogAspect.php [hunk 2]
```diff
@@ -37,6 +40,12 @@ class EventLogAspect
      */
     protected $nodeDataRepository;
 
+    /**
+     * @Flow\Inject(name="Neos.Flow:SystemLogger")
```
Comment (markusguenther) [inline Classes/Aspect/EventLogAspect.php]: And then use here just:

```
    /**
     * @Flow\Inject
     * @var SystemLoggerInterface
     */
    protected $logger;
```

**Review by markusguenther — APPROVED** [review]
Thanks for upgrading the package to the latest version 👏

**Pull request merged**